oneview/resource_rack_manager.go

```diff
@@ -0,0 +1,239 @@
+// (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+package oneview
+
+import (
+	"log"
+
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceRackManager() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceRackManagerCreate,
+		Read:   resourceRackManagerRead,
+		//Update: resourceRackManagerUpdate,
```

Comment:
what will happen as there is no update action? will Terraform invokes error saying no update